cinema/serializers.py

```diff
@@ -68,13 +77,62 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs=attrs)
+        Ticket.validate_ticket(
+            attrs["row"], attrs["seat"], attrs["movie_session"]
```

Comment:
The `validate_ticket` method is missing the `error_to_raise` argument. Ensure you pass the `ValidationError` class as the fourth argument to `validate_ticket`.